test/reviewController.test.js

```diff
@@ -124,4 +124,95 @@ describe('ReviewController API Tests', () => {
       expect(response.body.allProductReviews).toEqual([]);
     }
   });
+  //create a review
+  it('should create a review successfully', async () => {
+    await Review.deleteMany({});
+    const userCredentials = {
+      email: 'Emily@google.com',
+      password: 'secret',
+    };
+
+    const signedCookie = await loginAndReturnCookie(userCredentials);
+
+    const album = await Album.findOne({ albumName: 'Unique Album' });
```

Comment:
Since we store this album in a variable already (declared on line 29, and assigned in the `beforeAll` block on line 42), there's not really any need to go make a round-trip to the database to fetch it here in this test case. 

We can delete this line; and all the tests will still pass because we'll be referring to `album` that we created and stored in that variable earlier, without unnecessarily re-fetching it